_layouts/default.html

```diff
@@ -36,6 +36,11 @@
     <script src="//cdnjs.cloudflare.com/ajax/libs/jquery.matchHeight/0.7.0/jquery.matchHeight-min.js"></script>
     <script src="/dist/js/main.min.js"></script>
     <script>
+      $(document).ready(function(){
```

Comment:
I already moved this into a separate js file. (The js files gets concatenated and uglified.)
Two remarks:
- We should always wrap code like this into a [IIFE](https://en.wikipedia.org/wiki/Immediately-invoked_function_expression)s and pass `jQuery` to it as `$`. This way we stay compatible with the noConflict mode.
- Inline code is bad practice and one –  imho! – should only use inline js to pass initalization parameters (e.g. large backend generated JSON)
Lastly: I like the `data-toggle` initalization code!
https://github.com/dashpay/dash-website/pulls